llvm/test/CodeGen/AArch64/fp-maximumnum-minimumnum.ll

```diff
@@ -0,0 +1,190 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc --mtriple=aarch64 --mattr=+fullfp16 < %s | FileCheck %s --check-prefix=AARCH64
+
+define <2 x double> @max_nnan_v2f64(<2 x double> %a, <2 x double> %b) {
```

Comment:
Should test with and without nnan flag. Also should test some illegal vector widths, particularly 3 x and multiples of the legal vector sizes

Reply:
For cases without nnan, we cannot add them now, since they depends on fcanonicalize.
I will add them with https://github.com/llvm/llvm-project/pull/104429.

Reply:
For `multiples of the legal vector sizes`, I think that I have done them:
we have v4f64, v8f32, v16f16 here, and the length of NEON is 128bit.